modules/tf-cloud-credential/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "workspace_name_prefix" {
```

Comment:
This would probably be better as `name_prefix`.

Reply:
I resolved this in the latest commit for this PR